akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/Dialect.scala

```diff
@@ -51,7 +51,7 @@ private[projection] object DialectDefaults {
       "LAST_UPDATED" BIGINT NOT NULL
```

Comment:
You also have to delete the creation of the PK below and include it here in the conditional table creation:

```
    primary key ("PROJECTION_NAME", "PROJECTION_KEY")
```

Reply:
I will incorporate this. Thanks

Reply:
I don't remember why the PK is added on another statement. Anyway, if there is an issue we will find out when we change it.

Reply:
It certainly works for Postgress (current version) with the PK creation in the table creation.